ui/raidboss/data/04-sb/dungeon/doma_castle.txt

```diff
@@ -0,0 +1,137 @@
+112.3 "Magitek Ray" sync /:Rearguard Bit:20A1:/
+115.4 "Cermet Pile" sync /:Magitek Rearguard:209D:/
+118.4 "Magitek Ray" sync /:Rearguard Bit:20A1:/
+
+131.1 "Mines" sync /03:........:Added new combatant Rearguard Mine/ jump 68.3
+132.0 "Cermet Pile"
+135.0 "Magitek Ray"
+137.6 "Garlean Fire"
+143.1 "Magitek Ray"
+147.1 "Cermet Pile"
+150.1 "Magitek Ray"
+152.7 "Garlean Fire"
+158.2 "Magitek Ray"
+160.1 "Mines"
+
+### Magitek Hexadrone
+# -ii 20A4 20A6 20A7 2447
+1000 "Start" sync /00:0839:The Training Grounds will be sealed off/ window 1000,0
+1010.6 "Circle Of Death" sync /:Magitek Hexadrone:20A2:/
+1018.9 "2-Tonze Magitek Missile" sync /:Magitek Hexadrone:20A3:/
+1024.1 "Hexadrone Bits" sync /03:........:Added new combatant Hexadrone Bit/
+1029.6 "Bits Activate"
+1039.6 "Circle Of Death" sync /:Magitek Hexadrone:20A2:/
+1045.7 "Hexadrone Bits" sync /03:........:Added new combatant Hexadrone Bit/
+1051.2 "Bits Activate"
+1049.8 "Circle Of Death" sync /:Magitek Hexadrone:20A2:/
+1064.5 "Magitek Missiles" sync /:Magitek Hexadrone:20A5:/
+1068.9 "Hexadrone Bits" sync /03:........:Added new combatant Hexadrone Bit/
+1074.4 "Bits Activate"
+1077.3 "2-Tonze Magitek Missile" sync /:Magitek Hexadrone:20A3:/
+
+1083.9 "Circle Of Death" sync /:Magitek Hexadrone:20A2:/
+1090.0 "Hexadrone Bits" sync /03:........:Added new combatant Hexadrone Bit/
+1095.5 "Bits Activate"
+1098.6 "Magitek Missiles" sync /:Magitek Hexadrone:20A5:/
+1106.4 "2-Tonze Magitek Missile" sync /:Magitek Hexadrone:20A3:/
+1117.9 "Hexadrone Bits" sync /03:........:Added new combatant Hexadrone Bit/
+1123.4 "Bits Activate"
+1119.6 "Magitek Missiles" sync /:Magitek Hexadrone:20A5:/
+1132.3 "Circle Of Death" sync /:Magitek Hexadrone:20A2:/
+1138.7 "Hexadrone Bits" sync /03:........:Added new combatant Hexadrone Bit/
+1144.2 "Bits Activate"
+1146.7 "2-Tonze Magitek Missile" sync /:Magitek Hexadrone:20A3:/
+1156.8 "Magitek Missiles" sync /:Magitek Hexadrone:20A5:/
+1166.2 "Hexadrone Bits" sync /03:........:Added new combatant Hexadrone Bit/
+1171.7 "Bits Activate"
+1174.7 "2-Tonze Magitek Missile" sync /:Magitek Hexadrone:20A3:/
+
+1181.3 "Circle Of Death" sync /:Magitek Hexadrone:20A2:/ jump 1083.9
+1187.4 "Hexadrone Bits"
+1192.9 "Bits Activate"
+1195.0 "Magitek Missiles"
+1203.8 "2-Tonze Magitek Missile"
+1216.0 "Magitek Missiles"
+
+
+### Hypertuned Grynewaht
+# -ii 20A9 20AB 20AC 20AE 20A7 2447
+2000 "Start" sync /00:0839:The Hall Of The Scarlet Swallow will be sealed off/ window 2000,0
+2009.4 "Chainsaw" duration 5.5 sync /:Hypertuned Grynewaht:20A8:/
+2022.7 "Delay-Action Charge" sync /:Hypertuned Grynewaht:20AD:/
+2023.1 "Gunsaw" duration 5.0  sync /:Hypertuned Grynewaht:20AA:/
+2045.4 "Thermobaric Charge" sync /:Hypertuned Grynewaht:20AF:/
+2048.8 "--sync--" sync /:Hypertuned Grynewaht:20B0:/
+2057.0 "Gunsaw" duration 5.0  sync /:Hypertuned Grynewaht:20AA:/
+2070.3 "Delay-Action Charge/Clean Cut" sync /:Hypertuned Grynewaht:20AD:/
```

Comment:
Is there a different ability id for `Delay-Action Charge` and `Clean Cut`?

> The exact ordering on the simultaneous Delay-Action Charge and Clean Cut varies from pull to pull, so I just combined them in the timeline to avoid any missed syncs.

Is this one of those situations where he will either do one or the other mechanic, but it's a single ability and it happens at the same time regardless?  (Similar to Shield Omen / Sword Omen here https://github.com/quisquous/cactbot/blob/main/ui/raidboss/data/05-shb/eureka/delubrum_reginae.txt#L338.)

> The name is a little long, though, and Cut gets cut off with default timeline bar width. Any preference on abbreviation here, or even just leaving it as is?

In general, I'd suggest leaving the full ability names in the timeline itself.  This helps make translations easier because they can translate full ability names and not have to worry about English abbreviations.  You can add a `timelineReplace` section like https://github.com/quisquous/cactbot/blob/main/ui/raidboss/data/05-shb/eureka/delubrum_reginae.js#L1421-L1430 to replace them dynamically with something shorter.  Maybe, `Delayed Charge/Clean Cut` or something else?

Reply:
It's two abilities that happen at almost the same time (0.2 seconds apart, in variable order) In play, it's essentially a single mechanic resolution (dodge chakram ground AoEs while not stacking the Delay-Action player-AoEs) but it comes from two different actors and they aren't perfectly synced up.

Reply:
Oh! In that case, I'd add two separate timeline lines at the same time and comment out one of the syncs.  If one of the syncs is on `Grynewaht`, then I'd keep that one and comment out the other.  In general, other actors can definitely be inconsistent.